analysis.go

```diff
@@ -24,83 +27,22 @@ type Info struct {
 
 // Result is a result of a single check, it can have multiple Event and Info
 // objects attached to it.
+// REVIEW: why do we have YAML annotations?
```

Comment:
Depending on what you use to produce YAML, some libs look at the value for the `json` tag...
I propose getting rid of this?